sdk/notificationhubs/notification-hubs/review/notification-hubs.api.api.md

```diff
@@ -0,0 +1,99 @@
+## API Report File for "@azure/notification-hubs"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+/// <reference types="node" />
+
+import { ClientOptions } from '@azure-rest/core-client';
```

Comment:
We should talk about the filename... the nicest filename I found was `<packageName>.<subpath>.api.md`

That is lovely, for example you might see `notification-hubs.models.api.md` and know that `models` is the export

Where it's not great is here, with a `/api` subpath export 

Definitely open to other names if folks have ideas! let me know what you'd like to see

Reply:
The `/api` subpath export is going to make a regular appearance with the new modular codegen so this is definitely something we need to figure out.

I wouldn't mind using another dash e.g. `notification-hubs-api.api.md`. As long as it's sitting alongside the normal `notification-hubs.api.md` I don't think it's ambiguous, and it feels a _little_ easier to parse to me than `.api.api.md`. But I don't think there's much in it.

Reply:
Ya, I can go with `-` , will push that up shortly